packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -109,62 +115,52 @@ export default function ListViewSidebar( { listViewToggleElement } ) {
 		handleToggleListViewShortcut
 	);
 
-	/**
-	 * Render tab content for a given tab name.
-	 *
-	 * @param {string} tabName The name of the tab to render.
-	 */
-	function renderTabContent( tabName ) {
-		if ( tabName === 'list-view' ) {
-			return (
-				<div className="edit-post-editor__list-view-panel-content">
-					<ListView dropZoneElement={ dropZoneElement } />
-				</div>
-			);
-		}
-		return <ListViewOutline />;
-	}
-
 	return (
 		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
 		<div
 			className="edit-post-editor__document-overview-panel"
 			onKeyDown={ closeOnEscape }
 			ref={ sidebarRef }
 		>
-			<Button
-				className="edit-post-editor__document-overview-panel__close-button"
-				icon={ closeSmall }
-				label={ __( 'Close' ) }
-				onClick={ closeListView }
-			/>
-			<TabPanel
+			<div
```

Comment:
Is keeping this `div` around necessary?

Reply:
Hm... perhaps not! It looks like it's really only here to carry the ref! The style associated with this classname forces 100% height, but I think that's just to keep the div from collapsing and shrinking the drop zone.

The ref is used to handle the focus behavior when there aren't any blocks in the list view to focus. If we put that ref onto the `tablist` (and rename it!), that behavior should still work normally. We then don't need the `div`, the classname, or the related style. Nice, thank you!